src/main/java/nl/tudelft/ti2206/group9/entities/Player.java

```diff
@@ -99,13 +115,14 @@ public void setInvincible(final boolean set) {
 	@Override
 	public final void collision(final AbstractEntity collidee) {
 		if (collidee instanceof Coin) {
+			apCoin.play();
 			OBSERVABLE.notify(Category.PLAYER,
 					GameObserver.Player.COLLISION, Coin.class.getSimpleName());
```

Comment:
I have an idea. You have created AudioPlayers in Player, while this is something that the GUI should handle. What if you make a GameObserver that checks whether the Player is jumping/sliding/moving/etc, and play the sound whenever the Observer sends out an update? This probably works, because all AudioPlayers play sound whenever there is a message to the Observable.

<organization,review